spanner/transaction.js

```diff
@@ -126,8 +126,8 @@ function readWriteTransaction (instanceId, databaseId) {
           console.log(`The second album's marketing budget: ${secondBudget}`);
 
           // Makes sure the second album's budget is sufficient
-          if (secondBudget < transferAmount) {
-            throw new Error(`The second album's budget (${secondBudget}) is less than the transfer amount (${transferAmount}).`);
```

Comment:
Nit: can we make the `300000` a constant, and put its declaration next to `transferAmount`?

(I prefer having constants defined in one place if possible rather than repeating literal values.)